.gitignore

```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+/node_modules
+/test.md
```

Comment:
- [ OPTIONAL]  please note that to ignore a file in the .gitinore file, all you need to do is add the file name like this `test.md`.  The slash is added if you are ignoring a folder like `/node_modules`

Reply:
- [ ] Your description of the changes introduced in this PR is not clear. You can use bullet points to break down what you did ie
  - Add test.md to .gitignore
  - Add h1 text
  - Apply yellow color to h1

Reply:
@wayungi, Are you talking about the description in the readme file?